local-cli/core/index.js

```diff
@@ -70,11 +70,13 @@ const defaultConfig = {
   hasteImplModulePath: require.resolve('../../jest/hasteImpl'),
 
   getPlatforms(): Array<string> {
-    return ['ios', 'android', 'windows', 'web', 'dom'];
+    // if this is less than three items, AndroidTest stops resolving:
+    // Error: Unable to resolve module `ProgressBarAndroid` from `/Users/mhargett/workspace/react-native/ReactAndroid/src/androidTest/js/ProgressBarTestModule.js`: Module `ProgressBarAndroid` does not exist in the Haste module map
+    return ['ios', 'android', 'native', ...Object.keys(pluginPlatforms)];
   },
 
   getProvidesModuleNodeModules(): Array<string> {
-    return ['react-native', 'react-native-windows', 'react-native-dom'];
+    return ['react-native', ...plugins.platforms];
```

Comment:
`plugins.platforms` is an array of strings like this:

```javascript
[ 'react-native-foo/local-cli/platform.js' ]
```

Which it gets from an entry in `react-native-foo`'s `package.json` like this:

```javascript
  "rnpm": {
    "platform": "./local-cli/platform.js"
  }
```

But `getProvidesModuleNodeModules()` just wants entries like `react-native-foo`, so unless I did something wrong this doesn't seem to work. 

BTW, I just discovered this repo:

https://github.com/react-native-community/discussions-and-proposals/issues

Perhaps a discussion on how renderer plugins could be implemented could be started there too?

Reply:
Fixed. Yes, that would be a good place to propose a new API, or discuss on the contributor's slack.